x/stake/keeper/key.go

```diff
@@ -102,34 +102,34 @@ func GetTendermintUpdatesKey(ownerAddr sdk.AccAddress) []byte {
 
 //________________________________________________________________________________
 
-// get the key for delegator bond with validator.
+// GetDelegationKey Gets the key for delegator bond with validator.
 // VALUE: stake/types.Delegation
 func GetDelegationKey(delegatorAddr, validatorAddr sdk.AccAddress) []byte {
 	return append(GetDelegationsKey(delegatorAddr), validatorAddr.Bytes()...)
 }
 
-// get the prefix for a delegator for all validators
+// GetDelegationsKey Gets the prefix for a delegator for all validators
 func GetDelegationsKey(delegatorAddr sdk.AccAddress) []byte {
 	return append(DelegationKey, delegatorAddr.Bytes()...)
 }
 
 //________________________________________________________________________________
 
-// get the key for an unbonding delegation by delegator and validator addr.
+// GetUBDKey Gets the key for an unbonding delegation by delegator and validator addr.
 // VALUE: stake/types.UnbondingDelegation
 func GetUBDKey(delegatorAddr, validatorAddr sdk.AccAddress) []byte {
 	return append(
 		GetUBDsKey(delegatorAddr.Bytes()),
 		validatorAddr.Bytes()...)
 }
 
-// get the index-key for an unbonding delegation, stored by validator-index
+// GetUBDByValIndexKey Gets the index-key for an unbonding delegation, stored by validator-index
```

Comment:
Don't wanna be super nitpicky here, but can we make these complete sentences and wrap them (add a `.` and wrap at either 80 or 120 columns)?

Reply:
Ok np, should we make an standard on our comments for that ?